### PR TITLE
BPK-4314-Migrate-remaining-guideline-pages

### PR DESCRIPTION
**Pull request by olliecurtis**
This PR migrates the remaining guideline pages to the new MDX format.

**Changes**
- A bug fixed with the new table of contents implementation whereby any intro text was not displayed when following after a toc - so moved it to a new section called `Introduction`
- Created Typography and Colour components for use to display interactive components on the respective guideline pages.
- Deleted old js pages and routes

### CODE REVIEW

File: docs/src/components/Typography/TypeScale.js [hunk 1]
```diff
@@ -57,21 +57,18 @@ const formatSizes = sizes => {
...
-const TypeScale = props => {
-  const { className, ...rest } = props;
-
+const TypeScale = () => {
   return (
     <div
       aria-label="Type scale information"
-      className={getClassName('bpk-docs-type-scale', className)}
-      {...rest}
+      className={getClassName('bpk-docs-type-scale')}
```
Comment (olliecurtis) [inline docs/src/components/Typography/TypeScale.js:64]: Removed the overriding className and rest here as they weren't being used in the old implementation so added no value

File: docs/src/components/Typography/TypeScale.js [hunk 2]
```diff
@@ -57,21 +57,18 @@ const formatSizes = sizes => {
...
-      <div
-        aria-hidden="true"
-        className={getClassName('bpk-docs-type-scale__table')}
-      >
+      <div aria-hidden="true">
         {intervals.map(interval => (
-          <div className={getClassName('bpk-docs-type-scale__row')}>
+          <div
+            className={getClassName('bpk-docs-type-scale__row')}
+            key={interval.style}
```
Comment (olliecurtis) [inline docs/src/components/Typography/TypeScale.js:70]: Added a key as there was a dev console error complaining that each component wasn't unique

File: docs/src/components/Typography/font-face.scss [hunk 3]
```diff
@@ -21,12 +21,13 @@
...
   display: flex;
   width: max-content;
+  margin-bottom: $bpk-spacing-xxl;
   flex-direction: column;
 
   &__text {
     width: max-content;
-    font-size: $bpk-spacing-xxl * 2;
-    line-height: $bpk-spacing-xxl * 2;
+    font-size: $bpk-spacing-base * 2;
```
Comment (olliecurtis) [inline docs/src/components/Typography/font-face.scss:29]: Changed the sizing to not be as large and be able to display better on the page

File: docs/src/components/Typography/type-scale.scss [hunk 4]
```diff
@@ -41,10 +46,4 @@
       color: $bpk-color-sky-gray-tint-03;
     }
   }
-
-  &__table {
```
Comment (olliecurtis) [inline docs/src/components/Typography/type-scale.scss:45]: Removed as we no longer want to be overly complex and fancy. Something we should've originally pushed back on

File: docs/src/static-pages/logo.mdx [hunk 5]
```diff
@@ -6,6 +6,8 @@ category: guidelines
 
 ## Table of Contents
 
+## Introduction
```
Comment (olliecurtis) [inline docs/src/static-pages/logo.mdx:9]: As per my main comment in the PR this was added to fix an issue with the intro text not being displayed when ToC was being used

**Review by shaundon — DISMISSED** [review]

**Review by shaundon — APPROVED** [review]

**Comment by olliecurtis** [comment]
> olliecurtis deleted a comment from backpackbot now

This comment from Backpack bot was when I was fixing file renaming so can be ignored as it was fixed in the last commit

**Pull request merged**